app/src/main/java/com/allan/boardbuddies/fragments/NotesFragment.java

```diff
@@ -68,4 +78,49 @@ public void onClick(View view) {
             }
         });
     }
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        loadNotes();
```

Comment:
right now you're loading notes twice, once in onCreate and once here

Reply:
removing from oncreate